webrtc.html

```diff
@@ -7625,11 +7625,13 @@ <h3>Simulcast functionality</h3>
         <p>As defined in <span data-jsep="simulcast">[[!JSEP]]</span>, an offer from a user-agent will
         only contain a "send" description and no "recv" description on the <code class=sdp>a=simulcast</code> line.
         Alternatives and restrictions (described in [[MMUSIC-SIMULCAST]]) are not supported.</p>
-        <p>This specification does not define how to configure {{RTCPeerConnection/createOffer}} to receive multiple
-        RTP encodings. However when {{RTCPeerConnection/setRemoteDescription}} is called with a corresponding remote
-        description that is able to send multiple RTP encodings as defined in [[!JSEP]], the
-        {{RTCRtpReceiver}} may receive multiple RTP encodings and the parameters retrieved
-        via the transceiver's {{RTCRtpTransceiver/receiver}}.{{RTCRtpReceiver/getParameters()}} will reflect the encodings negotiated.</p>
+        <p>This specification does not define how to configure reception of multiple
+        RTP encodings using {{RTCPeerConnection/createOffer}}, {{RTCPeerConnection/createAnswer}} or
+        {{RTCPeerConnection/addTransceiver}}. However when {{RTCPeerConnection/setRemoteDescription}}
+        is called with a corresponding remote description that is able to send multiple RTP encodings
+        as defined in [[!JSEP]], the {{RTCRtpReceiver}} may receive multiple RTP encodings and the
```

Comment:
add at comma: ", and the browser supports receiving multiple RTP encodings",

Reply:
done.